### PR TITLE
OboeTester: add workload fader, remove benchmark

### PR DESCRIPTION
**Pull request by philburk**
This will make it easier to dial in a specific workload and also is more robust.

### CODE REVIEW

File: apps/OboeTester/app/src/main/java/com/mobileer/oboetester/DynamicWorkloadActivity.java [hunk 1]
```diff
@@ -37,7 +37,8 @@
  * Enable PerformanceHints (ADPF).
  */
 public class DynamicWorkloadActivity extends TestOutputActivityBase {
-    private static final double WORKLOAD_MAX = 500.0;
+    private static final int WORKLOAD_HIGH_MIN = 30;
+    private static final int WORKLOAD_HIGH_MAX = 150;
     public static final double LOAD_RECOVERY_HIGH = 1.0;
```
Comment (robertwu1) [inline apps/OboeTester/app/src/main/java/com/mobileer/oboetester/DynamicWorkloadActivity.java:42]: Should LOAD_RECOVERY_HIGH and LOAD_RECOVERY_LOW be so close to each other in terms of values?

File: apps/OboeTester/app/src/main/java/com/mobileer/oboetester/DynamicWorkloadActivity.java [hunk 2]
```diff
@@ -283,25 +259,26 @@ public void onClick(View view) {
...
         }
-        NativeEngine.setCpuAffinityMask(1 << defaultCpuAffinity);
+        NativeEngine.setCpuAffinityMask(defaultCpuAffinityMask);
 
         mMultiLineChart = (MultiLineChart) findViewById(R.id.multiline_chart);
         mMaxCpuLoadTrace = mMultiLineChart.createTrace("CPU", Color.RED,
                 0.0f, 2.0f);
         mWorkloadTrace = mMultiLineChart.createTrace("Work", Color.BLUE,
-                0.0f, (float)WORKLOAD_MAX);
+                0.0f, (1.2f * WORKLOAD_HIGH_MAX));
```
Comment (robertwu1) [inline apps/OboeTester/app/src/main/java/com/mobileer/oboetester/DynamicWorkloadActivity.java:272]: Why 1.2?

File: apps/OboeTester/app/src/main/java/com/mobileer/oboetester/DynamicWorkloadActivity.java [hunk 3]
```diff
@@ -323,6 +300,16 @@ public void onClick(View view) {
...
         });
 
+        if (mDynamicWorkloadView != null) {
+            mDynamicWorkloadView.setWorkloadReceiver((w) -> {
+                setWorkloadHigh(w);
+            });
+
+            mDynamicWorkloadView.setLabel("High Workload");
+            mDynamicWorkloadView.setRange(WORKLOAD_HIGH_MIN, WORKLOAD_HIGH_MAX);
+            mDynamicWorkloadView.setFaderNormalizedProgress(0.53);
```
Comment (robertwu1) [inline apps/OboeTester/app/src/main/java/com/mobileer/oboetester/DynamicWorkloadActivity.java:310]: Why .53?

File: apps/OboeTester/app/src/main/java/com/mobileer/oboetester/DynamicWorkloadActivity.java [hunk 4]
```diff
@@ -218,6 +186,11 @@ private void stop() {
 
     }
 
+    private void setWorkloadHigh(int workloadHigh) {
+        mWorkloadHigh = workloadHigh;
+        mWorkloadLow = Math.max(1, (int)(workloadHigh * 0.016));
```
Comment (robertwu1) [inline apps/OboeTester/app/src/main/java/com/mobileer/oboetester/DynamicWorkloadActivity.java:191]: Any reason why we have both .016 and LOAD_RECOVERY_HIGH?

File: apps/OboeTester/app/src/main/res/layout/activity_dynamic_workload.xml [hunk 5]
```diff
@@ -38,7 +38,7 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_marginRight="8sp"
-            android:text="Perf Hint" />
+            android:text="ADPF" />
```
Comment (robertwu1) [inline apps/OboeTester/app/src/main/res/layout/activity_dynamic_workload.xml:41]: Do we have a description of ADPF anywhere on the test?

File: apps/OboeTester/app/src/main/java/com/mobileer/oboetester/DynamicWorkloadActivity.java (hunk 1 above)
Comment (philburk) [reply apps/OboeTester/app/src/main/java/com/mobileer/oboetester/DynamicWorkloadActivity.java:42]: Yes, for hysteresis. I added comments.

File: apps/OboeTester/app/src/main/java/com/mobileer/oboetester/DynamicWorkloadActivity.java (hunk 2 above)
Comment (philburk) [reply apps/OboeTester/app/src/main/java/com/mobileer/oboetester/DynamicWorkloadActivity.java:272]: So I can show the CPU going above the highest workload. Arbitrary number based on tuning.

File: apps/OboeTester/app/src/main/java/com/mobileer/oboetester/DynamicWorkloadActivity.java (hunk 4 above)
Comment (philburk) [reply apps/OboeTester/app/src/main/java/com/mobileer/oboetester/DynamicWorkloadActivity.java:191]: Now I just set it to 1.

File: apps/OboeTester/app/src/main/res/layout/activity_dynamic_workload.xml (hunk 5 above)
Comment (philburk) [reply apps/OboeTester/app/src/main/res/layout/activity_dynamic_workload.xml:41]: I added some comments at the top of the file.

File: apps/OboeTester/app/src/main/java/com/mobileer/oboetester/DynamicWorkloadActivity.java (hunk 3 above)
Comment (philburk) [reply apps/OboeTester/app/src/main/java/com/mobileer/oboetester/DynamicWorkloadActivity.java:310]: Commented, for 70 voices.

**Review by robertwu1 — APPROVED** [review]

**Pull request merged**